gateway.go

```diff
@@ -51,9 +61,7 @@ func (s *gatewayResource) httpError(w http.ResponseWriter, status int, debugMess
 }
 
 func (s *gatewayResource) gatewayHandler(w http.ResponseWriter, r *http.Request) {
-	if s.verbose {
-		log.Printf("%s Handling %s\n", r.Method, r.URL.Path)
-	}
+	slog.Log(s.context, s.verboseLogLevel(), "Handling", "method", r.Method, "path", r.URL.Path)
```

Comment:
I think we should just replace the verbose semantics with something that fits better with modern structured logs. In particular, I'd just use `slog.Debug()` here (or whatever is appropriate).

Here and below.